bson/src/main/org/bson/codecs/pojo/PojoBuilderHelper.java

```diff
@@ -53,88 +46,104 @@
         }
```

Comment:
Unrelated note: the conditional just above here is not entered for any tests.  It is necessary?